terminal/bundles/org.eclipse.terminal.view.ui/src/org/eclipse/terminal/view/ui/IPreferenceKeys.java

```diff
@@ -71,8 +69,4 @@ public interface IPreferenceKeys {
 	public final String PREF_LOCAL_TERMINAL_DEFAULT_SHELL_UNIX_ARGS = PREF_TERMINAL
 			+ ".localTerminalDefaultShellUnixArgs"; //$NON-NLS-1$
 
-	static ScopedEclipsePreferences getPreferences() {
```

Comment:
If you remove this, the whole `IPreferenceKeys` becomes quite useless as there is no simple way without using internal API to ever access any of these keys.

Reply:
Activator is not really an API the method just redirects there (where one of course can unwind the call to the native ones to not reference it).